helpers.go

```diff
@@ -1,56 +1,60 @@
 package dbft
 
 type (
-	// inbox is a structure storing messages from a single epoch.
-	inbox[H Hash, A Address] struct {
-		prepare map[uint16]ConsensusPayload[H, A]
-		chViews map[uint16]ConsensusPayload[H, A]
-		commit  map[uint16]ConsensusPayload[H, A]
+	// Inbox is a structure storing messages from a single epoch.
+	Inbox[H Hash, A Address] struct {
```

Comment:
This I don't like. It's an internal thing. Does anything change wrt package size if you leave it as is (I know test code will have to import some implementation, but it's not compiled unless you do testing)?

Reply:
> This I don't like. It's an internal thing.

I completely understand, but otherwise it's required to implement the whole Payload/Hash/Address interfaces on new structs, I thought it doesn't worth the effort. Fixed.

> Does anything change wrt package size if you leave it as is

Yes, exported version is larger:
```
-rw-rw-r-- 1 anna anna 188828 мар  4 12:44 build.exported
-rw-rw-r-- 1 anna anna 188676 мар  4 12:44 build.private
```